com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/AdWrapper.kt

```diff
@@ -205,5 +206,15 @@ class AdWrapper(private val ad: HeliumAd) {
         fun wrap(ad: HeliumAd): AdWrapper {
             return AdWrapper(ad)
         }
+
+        fun runTaskOnUiThread(runnable: Runnable) {
+            UnityPlayer.currentActivity.runOnUiThread {
+                try {
+                    runnable.run()
+                } catch (ex: Exception) {
+                    Log.w(TAG, "Exception found when running on UI Thread: ${ex.message}")
+                }
+            }
+        }
```

Comment:
You can use Coroutine to call something on a specified thread. I see you can import Coroutine in `UnityBridge.kt`. Is that doable here too?

Reply:
Yes, it is. However, this is already how it had been setup in the past, this is just moving the pre-existing method into this file. This will be removed once the old APIs are deleted. This is already how it has been for a very long time.